fe/fe-core/src/main/java/com/starrocks/catalog/TabletStatMgr.java

```diff
@@ -369,6 +369,9 @@ private void sendTasks() {
         }
 
         private void waitResponse() {
+            if (responseList == null) {
```

Comment:
could you add a comment on when this will be null?

Reply:
Updated